src/iarc7_motion/iarc_tasks/height_recovery_task.py

```diff
@@ -15,7 +15,6 @@
                                     TaskAborted,
```

Comment:
This should be tested in the sim before merging because there are somewhat bigger changes here. running the `hit_roomba` abstract will test the height recovery task.

Reply:
the hit_roomba abstract worked in the sim when tested